src/classes/devices/difluidMicrobalance.ts

```diff
@@ -0,0 +1,172 @@
+import { PeripheralData } from './ble.types';
+import { BluetoothScale, SCALE_TIMER_COMMAND, Weight } from './bluetoothDevice';
+import { Logger } from './common/logger';
+
+declare var ble: any;
+export class DifluidMicrobalance extends BluetoothScale {
+  public static DEVICE_NAME = 'difluidmicrobalance';
+  public static SERVICE_UUID = '00EE';
+  public static CHAR_UUID = 'AA01';
+
+  protected override weight: Weight = {
+    actual: 0,
+    old: 0,
+    smoothed: 0,
+    oldSmoothed: 0,
+  };
+
+  private logger: Logger;
+  public scaleUnit = 'g';
+
+  constructor(data: PeripheralData) {
+    super(data);
+    this.logger = new Logger('DiFluid Scale');
+    this.connect();
+  }
+
+  public static test(device: any): boolean {
+    return (
+      device && device.name
+      // && this.DEVICE_NAME.includes(device.name)
```

Comment:
How is the DiFluid device beeing tested if its the real one, after this line is commented out?

Reply:
Added the check again.